src/mobot_client/models/phone_numbers.py

```diff
@@ -0,0 +1,25 @@
+#  Copyright (c) 2021 MobileCoin. All rights reserved.
+
+#  Copyright (c) 2021 MobileCoin. All rights reserved.
+from phonenumbers import PhoneNumber
+from phonenumber_field.modelfields import PhoneNumberField as BasePhoneNumberField
+
+
+class PhoneNumberWithRFC3966(PhoneNumber):
+    @property
+    def country_code_string(self) -> str:
+        return f"+{self.country_code}"
+
+    @property
+    def rfc3966(self) -> str:
+        return f"+{self.country_code}{self.national_number}"
```

Comment:
How is this different than using `as_e164`? I am not sure why the field needs to be overridden, seems like an over-complication to me.
https://github.com/stefanfoulis/django-phonenumber-field/blob/main/phonenumber_field/phonenumber.py#L60-L66

Reply:
Removed

Reply:
I think https://github.com/mobilecoinofficial/mobot/blob/86d0fc0bdf11725c4a1e68e7a45400f266edc55a/src/mobot_client/models/phone_numbers.py should entirely be removed, unless I am missing something